api/src/libs/healthCheck.ts

```diff
@@ -0,0 +1,46 @@
+import { LogWriterType } from "@libs/logWriter";
+import axios, { AxiosError, AxiosResponse } from "axios";
+
+type Status = "PASS" | "FAIL" | "ERROR";
+type StatusResult = Record<string, Status>;
+
+const healthCheckEndPoints: Record<string, string> = {
+  self: "self",
+  elevator: "dynamics/elevator",
+  fireSafety: "dynamics/fire-safety",
+  housing: "dynamics/housing",
+  dynamicsLicenseStatus: "dynamics/license-status",
+  webserviceLicenseStatus: "webservice/license-status",
+  webserviceFormation: "webservice/formation",
+};
+
+const url =
+  process.env.STAGE === "prod"
+    ? "https://api.navigator.business.nj.gov"
+    : "https://dev.api.navigator.business.nj.gov";
+
+const healthCheck = async (type: string, logger: LogWriterType): Promise<Status> => {
+  return axios
```

Comment:
[pebble] It's 2024; let's ditch axios for `fetch()` in all new development and slowly phase it out of our code base.